src/Microsoft.DotNet.Web.ProjectTemplates/content/StarterWeb-CSharp/Data/SqlLite/ApplicationDbContextModelSnapshot.cs

```diff
@@ -241,7 +240,8 @@ protected override void BuildTargetModel(ModelBuilder modelBuilder)
                         .HasForeignKey("UserId")
                         .OnDelete(DeleteBehavior.Cascade);
                 });
+
 #pragma warning restore 612, 618
-        }
+         }
```

Comment:
How did this extra space get here? The indentation is now off. Is this a bug in the migration generation logic itself?

Reply:
These current files are still the result of the previous hand merge/edits.  The next PR will add the batch files that can be used to generate these for check in purposes, and I'll regenerate the migrations with the new batch files.

Reply:
Ah OK so I'll ignore for now. Thanx!